packages/renderer/src/lib/kube/KubeApplyYAML.svelte

```diff
@@ -0,0 +1,254 @@
+          Create custom YAML from scratch
+        </div>
+      {/snippet}
+
+      {#snippet optionSnippet(option: 'file' | 'custom', label: string, content: Snippet)}
+        <button disabled={runStarted}
+          class="border-2 rounded-md p-5 cursor-pointer bg-(--pd-content-card-inset-bg)"
+          aria-label={label}
+          aria-pressed={userChoice === option ? 'true' : 'false'}
+          class:border-[var(--pd-content-card-border-selected)]={userChoice === option}
+          class:border-[var(--pd-content-card-border)]={userChoice !== option}
+          onclick={toggle.bind(undefined, option)}>
+          <div class="flex flex-row align-middle items-center">
+            <div
+              class="text-2xl pr-2"
+              class:text-[var(--pd-content-card-border-selected)]={userChoice === option}
+              class:text-[var(--pd-content-card-border)]={userChoice !== option}>
+              <Icon icon={faCircleCheck} />
+            </div>
+            {@render content()}
+          </div>
+        </button>
+      {/snippet}
+
+      {@render optionSnippet('file', '.yaml file to apply', file)} <!-- eslint-disable-line sonarjs/no-use-of-empty-return-value -->
+      {@render optionSnippet('custom', 'Custom yaml to apply', custom)} <!-- eslint-disable-line sonarjs/no-use-of-empty-return-value -->
+    </div>
+
+    <!-- Monaco Editor for custom YAML content -->
+    {#if userChoice === 'custom'}
+      <div class="space-y-3">
+        <label for="custom-yaml-editor" class="block text-base font-bold text-(--pd-content-card-header-text)">
+          Custom Kubernetes YAML Content
+        </label>
+        <div id="custom-yaml-editor" class="h-[400px] border">
+          <MonacoEditor
+            readOnly={runStarted}
+            language="yaml"
+            on:contentChange={(e): void => {
+              customYamlContent = e.detail;
+            }}/>
+        </div>
+      </div>
+    {/if}
+
+    {#if runStarted}
+      <div class="text-(--pd-content-card-text) text-sm">
+        Please don't leave the page while the Kubernetes YAML is being applied. This process may take a few minutes to complete...
+      </div>
+    {/if}
+
+    {#if runWarning}
+      <WarningMessage class="text-sm" error={runWarning} />
+    {/if}
+
+    {#if runError}
+      <ErrorMessage class="text-sm" error={runError} />
+    {/if}
+
+    {#if applyKubeResultRaw}
+      <div class="text-(--pd-content-card-text) text-sm">{applyKubeResultRaw}</div>
+    {/if}
+
+    <div class="grid grid-cols-[auto_auto] gap-4 justify-stretch">
+      <Button onclick={():void => router.goto(get(lastPage).path)} type="secondary">{runFinished? 'Done' : 'Cancel'}</Button>
+      <Button
+        type="primary"
+        onclick={kubeApply}
+        disabled={hasInvalidFields || runStarted}
+        inProgress={runStarted}
+        icon={KubeApplyIcon}>
+        {userChoice === 'custom' ? 'Apply Custom YAML' : 'Apply'}
+      </Button>
```

Comment:
issue: I don't expect to see the apply button once I already applied the resources

<img width="951" height="373" alt="Image" src="https://github.com/user-attachments/assets/c10ae470-0d28-4295-a22d-b7280b2698f1" />

At least this is not consistent with the `Podman Kube Play`

<img width="1079" height="332" alt="Image" src="https://github.com/user-attachments/assets/404d95f7-cd31-45bf-bb12-7e8809aa2913" />